### PR TITLE
Move to github actions

### PR DESCRIPTION
**Pull request by HeyNonster**
- Drops Travis
- Also drops Ruby < 2.7
- Tests with Ruby 2.7

Thanks for submitting a pull request to Koala! A huge portion of the gem has been built by awesome people (like you) from the Ruby community.

Here are a few things that will help get your pull request merged in quickly. None of this is required -- you can delete anything not relevant. If in doubt, open the PR! I want to help.

[ ] My PR has tests and they pass!
[ ] The live tests pass for my changes (`LIVE=true rspec` -- unrelated failures are okay).
[ ] The PR is based on the most recent master commit and has no merge conflicts.

If you have any questions, feel free to open an issue or comment on your PR!

Note: Koala has a [code of conduct](https://github.com/arsduo/koala/blob/master/code_of_conduct.md). Check it out.

### CODE REVIEW

File: spec/support/koala_test.rb [hunk 1]
```diff
@@ -235,7 +235,6 @@ def self.activate_adapter!
       # allow live tests with different adapters
       adapter = ENV['ADAPTER'] || "typhoeus" # use Typhoeus by default if available
       begin
-        # JRuby doesn't support typhoeus on Travis
```
Comment (bquorning) [inline spec/support/koala_test.rb:238]: Does JRuby support typhoeus on GitHub Actions CI?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+jobs:
+  specs:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version:
+          - "2.7"
```
Comment (bquorning) [inline .github/workflows/ci.yml:13]: This looks like overkill for testing just one Ruby version, but I suspect you want to test a few more in a subsequent PR?

**Review by bquorning — DISMISSED** [review]

**Comment by bquorning** [comment]
Perhaps we should instead consider adding the recent changes from the main fork of this repo: https://github.com/zendesk/koala/commit/bf030026e5b242a8de1bc83500bf637b6092be62

File: .github/workflows/ci.yml (hunk 2 above)
Comment (HeyNonster) [reply .github/workflows/ci.yml:13]: I do, yep!

**Comment by HeyNonster** [comment]
> Perhaps we should instead consider adding the recent changes from the main fork of this repo: [bf03002](https://github.com/zendesk/koala/commit/bf030026e5b242a8de1bc83500bf637b6092be62)

Hmm, probably.

**Comment by zenspider** [comment]
dead. closing

**Pull request closed without merging**